### PR TITLE
Raven db 8355

### PR DESCRIPTION
**Pull request by ml054**

### CODE REVIEW

File: src/Raven.Server/Documents/Handlers/Admin/AdminSortersHandler.cs [hunk 1]
```diff
@@ -53,6 +53,8 @@ public async Task Put()
 
                 await Database.RachisLogIndexNotifications.WaitForIndexNotification(index, ServerStore.Engine.OperationTimeout);
 
+                NoContentStatus();
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Handlers/Admin/AdminSortersHandler.cs:56]: We are setting 'Created' 2 lines below?

File: src/Raven.Server/Documents/Handlers/SortersHandler.cs [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+                var sorters = record.Sorters;
+                
+                using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
+                {
+                    
+                    writer.WriteStartObject();
+
+                    if (sorters != null)
+                    {
+                        writer.WriteArray(context, "Sorters", sorters.Values, (w, c, sorter) =>
```
Comment (ppekrol) [inline src/Raven.Server/Documents/Handlers/SortersHandler.cs]: we want to have a property `Sorters` always, in worst case with empty array

File: src/Raven.Server/Documents/Handlers/Admin/AdminSortersHandler.cs (hunk 1 above)
Comment (ml054) [reply src/Raven.Server/Documents/Handlers/Admin/AdminSortersHandler.cs:56]: no content also removed application/json. It is common pattern in controllers. Although the method name is a bit misleading.

File: src/Raven.Server/Documents/Handlers/SortersHandler.cs (hunk 2 above)
Comment (ml054) [reply src/Raven.Server/Documents/Handlers/SortersHandler.cs]: done

**Pull request merged**